src/components/FieldDateSelector/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { FieldProps, useField } from '@formiz/core';
+
+import {
+  DateSelector,
+  DateSelectorNextDayButton,
+  DateSelectorPicker,
+  DateSelectorPreviousDayButton,
+} from '@/components/DateSelector';
+import { FormGroup, FormGroupProps } from '@/components/FormGroup';
+import dayjs, { Dayjs } from 'dayjs';
+
+export type FieldDateSelectorProps = FieldProps & FormGroupProps;
+
+export const FieldDateSelector = (props: FieldDateSelectorProps) => {
+  const { ...fieldProps } = props;
+  const {
+    errorMessage,
+    id,
+    isValid,
+    isSubmitted,
+    isPristine,
+    setValue,
+    value,
+    otherProps,
+  } = useField({
+    debounce: 0,
+    ...fieldProps,
+  });
+  const { children, label, placeholder, helper, size, ...rest } =
+    otherProps as Omit<FieldDateSelectorProps, keyof FieldProps>;
+  const { required } = props;
+  const showError = !isValid && (!isPristine || isSubmitted);
+
+  const formGroupProps = {
+    errorMessage,
+    helper,
+    id,
+    isRequired: !!required,
+    label,
+    showError,
+    ...rest,
+  };
+
+  function dayjsToDate(dayjsValue : Dayjs){
+    setValue(dayjs(dayjsValue)) 
+  }
```

Comment:
Prettier does not seems to be configured properly. If you are using VScode, you should install the recommended extensions (see `.vscode/extensions.json`).

If prettier was configured properly, the code should look like 

```suggestion
  function dayjsToDate(dayjsValue: Dayjs) {
    setValue(dayjs(dayjsValue));
  }
```